cg/apps/janus/api.py

```diff
@@ -0,0 +1,28 @@
+"""The JanusAPIClient."""
+
+from http import HTTPStatus
+
+import requests
+from requests import Response
+
+from cg.apps.janus.dto.create_collect_qc_request import CreateCollectQCRequest
+from cg.exc import JanusAPIError
+
+
+class JanusAPIClient:
+    def __init__(self, config: dict[str]):
+        self.host = config["janus"]["host"]
+
+    def get_qc_metrics(self, collect_qc_request: CreateCollectQCRequest):
+        endpoint: str = f"{self.host}/collect_qc_metrics"
```

Comment:
Consider renaming this endpoint to follow [RESTful design](https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design#define-api-operations-in-terms-of-http-methods). 

Following REST standards, the endpoint would be a noun (like `/qc_metrics`). The action, which is to create qc metrics, is indicated by the type of request you send (`POST` here) and should not be specified in the name of the endpoint

Reply:
fixed that